.github/scripts/test.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+set -eux
+
+. .github/scripts/prereq.sh
+. .github/scripts/common.sh
```

Comment:
Again, maybe we should prefer 
```suggestion
source .github/scripts/prereq.sh
source .github/scripts/common.sh
```

but maybe I am misguided

Reply:
Misguided :D